aodntools/timeseries_products/Documentation/aggregated_timeseries.md

```diff
@@ -0,0 +1,175 @@
+# Non-Velocity Aggregated Product
+
+- [Objective](#objective)
+- [Scope](#scope)
+- [Input](#input)
+- [Output](#output)
+- [Method](#method)
+- [Distribution](#distribution)
+
+
+
+## Objective
+
+This product project will provide:
+
+A comprehensive set of aggregated time-series files for each mooring site and parameter (excluding current velocity), 
+without any interpolation or filtering, except for the exclusion of the out-of-water data. The Quality Control (QC) 
+flags will be preserved. All the (python) code used for the generation of the products openly available on GitHub.
+Clear documentation on the code and the output file format.
```

Comment:
This file is the documentation, and it's here with the code on GitHub, so the last two sentences are a bit redundant.

Also, here we are documenting the _code_, not the project. So maybe instead of objective, the first thing to describe is the _output_, which is kind of what the first two sentences say (without the "comprehensive set.." bit).